build/checkstyle/suppressions.xml

```diff
@@ -10,4 +10,7 @@
     <suppress files="[\\/]src[\\/]main[\\/].*|[\\/]generated-sources[\\/].*" id="TestScope" />
     <!--  Suppress specific file  -->
     <suppress files="hub/server/src/main/java/alluxio/hub/manager/process/ManagerProcessContext.java" checks="Indentation" />
+    <!-- Excludes test files from having Javadocs for classes and methods (a lot do not have those) -->
+    <suppress files="[\\/]*[\\/]test[\\/].*" checks="MissingJavadocMethod" />
```

Comment:
maybe this needs to be `Test` instead of `test`?
also would it be more precise to check for files matching `*Test.java`? dunno if the checkstyle file matcher takes into account the file extension or not

Reply:
this is meant to escape all files who have a `test` parent directory, which would also include test utils and helpers. I think this is consistent with the old rule because when escaping only `*Test.java` file some violations occur.

Reply:
got it